routes/equations.js

```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+
+import verifyToken from '../middlewares/tokenMiddleware';
+import adminMiddleware from '../middlewares/adminMiddleware';
+
+import equationController from '../controllers/equation';
+
+const router = express.Router();
+
+router.post('/', verifyToken, adminMiddleware, (req, res, next) =>
+  equationController.addEquation(req, res, next),
+);
+router.get('/', (req, res, next) =>
+  equationController.getRandomEquation(req, res, next),
+);
+router.delete('/', verifyToken, adminMiddleware, (req, res, next) =>
```

Comment:
Please pass the id of the entity you want to delete in the url (`/:id`). Same for update